src/main/java/com/stock/stocklist/mapper/StockListMapper.java

```diff
@@ -29,4 +30,7 @@ public interface StockListMapper {
     @Update("update stock_list set name = #{name}, grade = #{grade}, quantity = #{quantity}, unit = #{unit}, purchase = #{purchase} where id = #{id}")
     void update(StockList stockList);
 
+    @Delete("delete from stock_list WHERE id = #{id}")
```

Comment:
WHEREを大文字にするならMySQLの用語は大文字にしたいです。

```suggestion
    @Delete("DELETE FROM stock_list WHERE id = #{id}")
```

Reply:
fcf790c5c60cd7b1ddf84c3be078cfc03e208017
全体的に大文字に修正しました。
ご確認お願いいたします。

Reply:
👍